jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/jsonb/JsonbDiagnosticsCollector.java

```diff
@@ -124,4 +138,36 @@ private void addJsonbTransientSpecificDiagnostics(Diagnostic d) {
         d.setMessage(JsonbConstants.ERROR_MESSAGE_JSONB_TRANSIENT);
         d.setCode(JsonbConstants.DIAGNOSTIC_CODE_ANNOTATION_TRANSIENT_FIELD);
     }
+    
+    private boolean hasJsonbTransient(IAnnotatable annotable) throws JavaModelException {
+        for (IAnnotation annotation : annotable.getAnnotations()) {
+            String annotationName = annotation.getElementName();
+            if (annotationName.equals(JsonbConstants.JSONB_TRANSIENT))
+                return true;
+        }
+        return false;
+    }
+    
+    private boolean hasOtherJsonbAnnotation(IAnnotatable annotable) throws JavaModelException {
```

Comment:
Nitpick: maybe a slightly better name would be `hasJsonbAnnotationOtherThanTransient` or `hasNonJsonbTransientAnnotation`, this may be more informative of the method to a lay reader. Though I don't feel too strongly about this and we could continue with the correct name based on your judgment.

Reply:
Fixed